react/week3/todolist/src/components/App.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import Form from './Form';
+import TodoList from './TodoList';
+import uuid from 'uuid';
+
+class App extends React.Component {
+   constructor() {
+      super();
+      this.state = {
+         isLoading: true,
+         todos: [],
+         id: uuid(),
+         done: true,
+         error: null,
+      };
```

Comment:
In the version of react we're using, you can omit the constructor and super, it's done behind the scenes, so you could re-write this like: 
```suggestion
      state = {
         isLoading: true,
         todos: [],
         id: uuid(),
         done: true,
         error: null,
      };
```
You can read more about it in the [docs](https://reactjs.org/docs/react-component.html#constructor), however I will admit, they are a bit confusing to understand.